apps/gallery/test/marionette/lib/gallery.js

```diff
@@ -23,41 +23,21 @@ Gallery.ORIGIN = 'app://gallery.gaiamobile.org';
  */
 Gallery.Selector = Object.freeze({
   thumbnail: '.thumbnail',
-  fullscreenView: '#fullscreen-view',
   thumbnailsView: '#thumbnail-views > footer.thumbnails-list',
   thumbnailsSelectButton: '#thumbnails-select-button',
   thumbnailsDeleteButton: '#thumbnails-delete-button',
-  fullscreenBackButton: '#fullscreen-back-button-tiny',
-  editButton: '#fullscreen-edit-button-tiny',
-  shareButton: '#fullscreen-share-button-tiny',
   confirmButton: '#confirm-ok',
   overlayView: '#overlay',
   overlayTitle: '#overlay-title',
   overlayText: '#overlay-text',
   overlayCameraButton: '#overlay-camera-button',
-  editView: '#edit-view',
-  editExposureButton: '#edit-exposure-button',
-  editCropButton: '#edit-crop-button',
-  editEffectButton: '#edit-effect-button',
-  editEnhanceButton: '#edit-enhance-button',
-  exposureOptions: '#exposure-slider',
-  cropOptions: '#edit-crop-options',
-  effectOptions: '#edit-effect-options',
-  exposureSlider: '#sliderthumb',
-  editCropAspectPortraitButton: '#edit-crop-aspect-portrait',
-  editEffectSepiaButton: '#edit-effect-sepia',
-  editSaveButton: '#edit-save-button',
-  editToolApplyButton: '#edit-tool-apply-button',
-  editHeader: '#edit-view gaia-header',
-  fullscreenFrame2: '#frame2',
-  fullscreenFrame3: '#frame3',
-  cropDoneButton: '#crop-done-button',
-  editCropCanvas: '#edit-crop-canvas',
   openTitle: '#filename',
   openImage: '#frame > .image-view',
   openSaveButton: '#save'
 });
 
+var Fullscreen_View = require('./fullscreen_view.js');
+
```

Comment:
This is confusing. If we can require('./fullscreen_view.js') and instantiate fullscreenview in tests , it will be a clean separation of gallery.js from fullscreen_view.js